src/openforms/submissions/tests/test_submission_suspension.py

```diff
@@ -218,3 +219,98 @@ def test_resume_url_does_not_work_after_submission_has_been_completed(self):
         # Validate the link no longer works
         response = self.client.get(resume_path)
         self.assertEqual(response.status_code, 403)
+
+
+class SuspendedSubmissionAPITest(SubmissionsMixin, APITestCase):
+    def setUp(self):
+        # clear caches to avoid problems with throttling; @override_settings not working
+        # see https://github.com/encode/django-rest-framework/issues/6030
+        self.addCleanup(clear_caches)
+        super().setUp()
+
+    def test_api_lists_suspended_submissions_for_bsn(self):
+        bsn = "123456789"
+        good_submissions = [
+            # SubmissionFactory.create(
+            #     uuid='aaaaaaaa-0001-aaaa-aaaa-aaaaaaaaaaaa',
+            #     auth_info__value=bsn,
+            #     auth_info__attribute=AuthAttribute.bsn,
+            #     # TODO fix 'suspended' factory trait to work like '_suspend' viewset action
+            #     suspended=True,
+            # ),
+            SubmissionFactory.create(
+                uuid="aaaaaaaa-0002-aaaa-aaaa-aaaaaaaaaaaa",
+                suspended_on=timezone.now(),
+                completed_on=None,
+                auth_info__value=bsn,
+                auth_info__attribute=AuthAttribute.bsn,
+            ),
+        ]
+        bad_submissions = [  # noqa F841
+            SubmissionFactory.create(
+                form__name="form completed",
+                completed=True,
+                auth_info__value=bsn,
+                auth_info__attribute=AuthAttribute.bsn,
+            ),
+            SubmissionFactory.create(
+                form__name="not our BSN",
+                suspended_on=timezone.now(),
+                completed_on=None,
+                auth_info__value="999999999",
+                auth_info__attribute=AuthAttribute.bsn,
+            ),
+            SubmissionFactory.create(
+                form__name="bad auth attributes",
+                suspended_on=timezone.now(),
+                completed_on=None,
+                auth_info__value=bsn,
+                auth_info__attribute=AuthAttribute.kvk,
+            ),
+            SubmissionFactory.create(
+                form__name="invalid dates",
+                suspended_on=timezone.now(),
+                completed_on=timezone.now(),
+                auth_info__value=bsn,
+                auth_info__attribute=AuthAttribute.bsn,
+            ),
+            SubmissionFactory.create(
+                form__name="null dates",
+                suspended_on=None,
+                completed_on=None,
+                auth_info__value=bsn,
+                auth_info__attribute=AuthAttribute.bsn,
+            ),
+            SubmissionFactory.create(
+                form__name="hashed BSN",
+                suspended_on=None,
+                completed_on=None,
+                auth_info__value="<!>hashed-bsn<!>",
+                auth_info__attribute=AuthAttribute.bsn,
+                auth_info__attribute_hashed=True,
```

Comment:
shortcut is available via `SubmissionFactory.create(auth_info__with_hashed_identifying_attributes=True)` trait